lib/tests/streamlit/delta_generator_test.py

```diff
@@ -91,88 +92,21 @@ def test_public_api(self):
             for name, _ in inspect.getmembers(DeltaGenerator)
             if not name.startswith("_")
         }
-        self.assertEqual(
-            api,
-            {
-                "add_rows",
-                "altair_chart",
-                "area_chart",
-                "audio",
-                "balloons",
-                "bar_chart",
-                "bokeh_chart",
-                "button",
-                "camera_input",
-                "caption",
-                "chat_input",
-                "chat_message",
-                "checkbox",
-                "code",
-                "color_picker",
-                "columns",
-                "container",
-                "dataframe",
-                "data_editor",
-                "date_input",
-                "dg",
-                "divider",
-                "download_button",
-                "empty",
-                "error",
-                "exception",
-                "expander",
-                "experimental_audio_input",
-                "feedback",
-                "file_uploader",
-                "form",
-                "form_submit_button",
-                "graphviz_chart",
-                "header",
-                "help",
-                "html",
-                "id",
-                "image",
-                "info",
-                "json",
-                "latex",
-                "line_chart",
-                "link_button",
-                "map",
-                "markdown",
-                "metric",
-                "multiselect",
-                "number_input",
-                "page_link",
-                "plotly_chart",
-                "popover",
-                "progress",
-                "pydeck_chart",
-                "pyplot",
-                "radio",
-                "scatter_chart",
-                "select_slider",
-                "selectbox",
-                "slider",
-                "snow",
-                "subheader",
-                "success",
-                "status",
-                "table",
-                "tabs",
-                "text",
-                "text_area",
-                "text_input",
-                "time_input",
-                "title",
-                "toast",
-                "toggle",
-                "vega_lite_chart",
-                "video",
-                "warning",
-                "write",
-                "write_stream",
-            },
-        )
+        expected_api = ELEMENT_COMMANDS.copy()
+
+        # Remove commands that are only exposed in the top-level namespace:
```

Comment:
It might be helpful to elaborate on what this means, maybe by adding ", so which cannot be called on the delta_generator" or similar.

Reply:
Added 👍